packages/mermaid/src/docs/syntax/flowchart.md

```diff
@@ -643,8 +651,10 @@ Available curve styles include `basis`, `bumpX`, `bumpY`, `cardinal`, `catmullRo
 In this example, a left-to-right graph uses the `stepBefore` curve style:
 
 ```
+
 %%{ init: { 'flowchart': { 'curve': 'stepBefore' } } }%%
 graph LR
+
```

Comment:
Hmmmm, I'm not sure what happened here to add these random newlines? Maybe a bad merge or some weird auto-formatting?

```suggestion
%%{ init: { 'flowchart': { 'curve': 'stepBefore' } } }%%
graph LR
```